domains/rootcircle.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rootCircle",
+        "email": "dev.frolics@gmail.com"
+    },
+    "record": {
+        "CNAME": "rootcircle.github.io"
```

Comment:
Also, I am not sure atm, that if I should use `rootcircle.github.io` or `rootcircle.github.io/blog` as CNAME. The former redirects to `rootcircle.github.io/blog`.

Can you please help me with that?

Reply:
You can't have `/` in `CNAME` so when people visits rootcircle.is-a.dev they'll be redirected to rootcircle.is-a.dev/blog

Reply:
okay cool! thanks, this too will work :-)